org.osgi.test.junit5.cm/test.bndrun

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) OSGi Alliance (2020). All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+-tester: biz.aQute.tester.junit-platform
+
+-runvm: ${def;argLine}
+
+-runfw: org.eclipse.osgi
+-resolve.effective: active
+-runproperties: \
+	logback.configurationFile=file:${.}/logback.xml,\
+	org.apache.felix.http.host=localhost,\
+	org.osgi.service.http.port=*,\
+	org.osgi.framework.bootdelegation=sun.reflect,\
+	osgi.console=
+	
+-runsystempackages: \
+	org.slf4j;version=1.7.25,\
+	org.slf4j.helpers;version=1.7.25,\
+	org.slf4j.spi;version=1.7.25
+-runpath: \
+	ch.qos.logback.classic,\
+	ch.qos.logback.core,\
+	org.apache.felix.logback,\
+	slf4j.api
+-runrequires: \
+	bnd.identity;id='${project.artifactId}-tests',\
+	bnd.identity;id='junit-jupiter-engine',\
+	bnd.identity;id='junit-platform-launcher'
+# Please try and keep this sorted to make diffing more consistent
+-runbundles: \
```

Comment:
Please sort the runbundles list. We try to keep it in sorted order so that we can easily see what changes.